src/rpc/rawtransaction.cpp

```diff
@@ -208,10 +207,8 @@ static UniValue gettxoutproof(const JSONRPCRequest& request)
         throw std::runtime_error(
             RPCHelpMan{"gettxoutproof",
                 "\nReturns a hex-encoded proof that \"txid\" was included in a block.\n"
-                "\nNOTE: By default this function only works sometimes. This is when there is an\n"
-                "unspent output in the utxo for this transaction. To make it always work,\n"
-                "you need to maintain a transaction index, using the -txindex command line option or\n"
-                "specify the block in which the transaction is included manually (by blockhash).\n",
+                "You must either manually specify the blockhash of the block including the transaction\n"
```

Comment:
This should be in the follow-up PR (for changing gettxoutproof)

Reply:
Doesn't this functionality change with this PR because `gettxoutproof` uses `GetTransaction` which is no longer checking the unspent utxo set as per the changes here: https://github.com/bitcoin/bitcoin/pull/15159/files/94af589358f2e1ede359263494535cda67313c8d#diff-24efdb00bfbe56b140fb006b562cc70bL1014

Reply:
Doesn't this functionality change with this PR because gettxoutproof uses GetTransaction which is no longer checking the unspent utxo set as per the changes here: https://github.com/bitcoin/bitcoin/pull/15159/files/94af589358f2e1ede359263494535cda67313c8d#diff-24efdb00bfbe56b140fb006b562cc70bL1014

Reply:
ahhh. gotcha